learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -143,21 +162,29 @@ public distinct service class CovidData {
 }
 ```
 
-Here, you are creating the `CovidData` service type to represent the GraphQL `Object` type that represents an entry in
-the data set. Each resource method in this service represents a field of the GraphQL object type.  The return type of
+In this code:
+- The endpoint returns the number of cases in thousands. Therefore, a service type is used to define the output object type, and inside the service type, each resource function will return the original value
+divided by 1000.
+- You are creating the `CovidData` service type to represent the GraphQL `Object` type, which represents an entry in
```

Comment:
Why only this point is started as `You are`?

Reply:
Changed to `The `CovidData` service type represents the GraphQL `Object` type, which represents an entry in
the data set. `